packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -72,11 +72,10 @@ export class SelectionZone extends BaseComponent<ISelectionZoneProps, {}> {
 
   public componentDidMount(): void {
     const win = getWindow(this._root.current);
-    const scrollElement = findScrollableParent(this._root.current);
 
     // Track the latest modifier keys globally.
     this._events.on(win, 'keydown, keyup', this._updateModifiers, true);
-    this._events.on(scrollElement, 'click', this._tryClearOnEmptyClick);
+    this._events.on(document, 'click', this._findScrollParentAndTryClearOnEmptyClick);
```

Comment:
This is potentially a behavior change.

Flagging @ThomasMichon for thoughts.

Reply:
Also Outlook just complained about document level or window level events (see keydown, keyup), because they effect the entire app.

E.g. Now when you click in editing an email, this helper will execute even though the SelectionZone is used on an unrelated part of the UX.

Reply:
Flagging @JiuqingSong fyi

Reply:
I'd prefer to mount it on some known node, but the findScrollableParent traversal means that we can't know for sure how far up the tree we're going to need to walk.

The only workarounds I can think of would be to expand the SelectionZone API to have either
1. a callback to let the parent bind the event
2. a RefObject parameter used to bind to bind during first mount.

(1.) seems like the better of the those 2 options (otherwise we enter react lifecycle method order hell), but I'm curious if you've thought of another possible approach that I'm missing? if unsupplied we could bind in componentDidMount as we currently do on `master`.

And sorry for the long turnaround! We just had FHL so our dev effort has been going towards that for the past week.

Reply:
no problem! Totally get FHL :)

Reply:
I talked to @JiuqingSong offline about this and he said that he's fine with the global listener in the current approach, since his main concern with global listeners is that they interfere with typing performance in OWA compose.

I feel like (1.) is still a reasonable option optimization option to provide to consumers since it side-steps the need to read the DOM at all, but has the possibility of adding incorrect behaviour (since the element your callback binds it to might not be the same as the scrollable element we find during traversal).

I'm leaning towards merging this as-is and doing (1.) as a follow-up enhancement. Let me know your thoughts and I'll go ahead with that path?